bool AS_active
uint8 AS_mission
bool AS_R2D
uint8 ASSI